### PR TITLE
deploy `VeJoeToken`, `VeJoeStaking` and dummy token for rewards

### PR DESCRIPTION
**Pull request by jummy123**

### CODE REVIEW

File: deploy/VeJoeStaking.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  let joeAddress,
+    veJoeAddress = (await deployments.get("VeJoeToken")).address,
+    baseGenerationRate = 0,
+    boostedGenerationRate = 1,
+    boostedThreshold = 1,
+    boostedDuration = 0,
+    maxCap = 1;
```
Comment (0xnivek) [inline deploy/VeJoeStaking.js:11]: we should confirm what these values should be, i'll ask in the staking channel

**Review by 0xnivek — COMMENTED** [review]
LGTM!

File: deploy/VeJoeStaking.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+module.exports = async function ({ getNamedAccounts, deployments }) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  let joeAddress,
+    veJoeAddress = (await deployments.get("VeJoeToken")).address,
+    baseGenerationRate = 0,
+    boostedGenerationRate = 1,
+    boostedThreshold = 1,
+    boostedDuration = 0,
```
Comment (0xnivek) [inline deploy/VeJoeStaking.js:10]: we've renamed these from `boosted` -> `speedUp` in the contract, let's reflect that here

Comment (0xnivek) [inline deploy/VeJoeStaking.js:10]: two small things on naming:
- we've renamed these from `boosted` -> `speedUp` in the contract, let's reflect that here.
- also rename `baseGenerationRate` -> `veJoePerSharePerSec` and `boostedGenerationRate` -> `speedUpVeJoePerSharePerSec`

**Review by LouisMeMyself — APPROVED** [review]
LGTM, I approve @0xnivek comments

File: deploy/VeJoeStaking.js (hunk 2 above)
Comment (jummy123) [reply deploy/VeJoeStaking.js:10]: Thanks for this comment, I will rebase this once latest `VeJoeStaking` changes are merged and update variables.

File: deploy/VeJoeStaking.js (hunk 1 above)
Comment (0xnivek) [reply deploy/VeJoeStaking.js:11]: ok from speaking w/ murloc, it seems like these values should be:
```
veJoePerSharePerSec = 3.170979198376e12
speedUpVeJoePerSharePerSec = 3.170979198376e12
speedUpThreshold = 5
speedUpDuration = 15 days
maxCapPct = 10000
```

File: deploy/VeJoeStaking.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+module.exports = async function ({ getNamedAccounts, deployments }) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  let joeAddress,
+    veJoeAddress = (await deployments.get("VeJoeToken")).address,
+    veJoePerSharePerSec = 3170979198376,
+    speedUpVeJoePerSharePerSec = 3170979198376,
```
Comment (cryptofish7) [inline deploy/VeJoeStaking.js:8]: Can you explain how `3170979198376` came about?

**Review by cryptofish7 — CHANGES_REQUESTED** [review]

**Review by cryptofish7 — APPROVED** [review]

**Pull request merged**